### PR TITLE
feat: added video and audio links

### LINKED ISSUE #1584: Adding video and audio links
**Actual Behaviour**

Currently we are not adding audio and video links which might be useful ahead.

**Expected Behaviour**

Adding the video and audio links.

**Steps to reproduce it**

-

**LogCat for the issue**

-

**Screenshots of the issue**

-

**Would you like to work on the issue?**

yes.

**Target Milestone to solve the issue**

https://github.com/fossasia/open-event-android/milestone/26

### PR DESCRIPTION
**Pull request by arp95**
Fixes #1584 

Changes: added video and audio links

Screenshots for the change: 

-

### CODE REVIEW

**Comment by arp95** [comment]
@iamareebjamal
as you are adding realm in the meantime, didnt make database changes right now.

File: android/app/src/main/java/org/fossasia/openevent/data/Session.java [hunk 1]
```diff
@@ -85,6 +90,22 @@ public void setTitle(String title) {
         this.title = title;
     }
 
+    public void setAudio_link(String audio_link){
```
Comment (Shailesh351) [inline android/app/src/main/java/org/fossasia/openevent/data/Session.java]: Hey,  keep method names in camel case

**Comment by iamareebjamal** [comment]
@arp95 It is already added in the Realm Implementation

**Comment by arp95** [comment]
@iamareebjamal
really sorry for not seeing that. as its better to keep stuff in one pr closing this then.

**Pull request closed without merging**